File: solutions/javascript/raindrops/1/raindrops.js
export const convert = num => {
  let factors = [];
  let raindropSpeak = "";
  for (let i =1;i< num+1; i++){
    if(num % i === 0) factors.push(i)
  }

  if (!factors.includes(3) && !factors.includes(5) && !factors.includes(7)) return "" + num;

  if (factors.includes(3)){
    raindropSpeak += "Pling";
  }
  if (factors.includes(5)){
    raindropSpeak += "Plang";
  }
  if (factors.includes(7)){
    raindropSpeak += "Plong";
  }

  return raindropSpeak;
};

File: solutions/python/raindrops/1/raindrops.py
def convert(number):
    factors = []
    for i in range(1, number+1):
        if number % i == 0:
            factors.append(i)

    NOT_IN_LIST = (3 not in factors and 5 not in factors and 7 not in factors)
    raindropSpeak = ""
    if NOT_IN_LIST:
        raindropSpeak += str(number)
    else:
        if 3 in factors:
            raindropSpeak += "Pling"
        if 5 in factors:
            raindropSpeak += "Plang"
        if 7 in factors:
            raindropSpeak += "Plong"

    return raindropSpeak